docs/docs/guides/troubleshooting.md

```diff
@@ -73,3 +73,37 @@ or
 ```bash
 npm install
 ```
+
+### Automatic workletization of gesture callbacks
+
+Reanimated's Babel plugin is setup in a way that automatically marks callbacks passed to gestures in the configuration chain as worklets. This means that as long as all your callbacks are defined in a single chain, you don't need to add a `'worklet';` directive at the beginning of the functions. Here is an example that will be automatically workletized:
+
+```jsx
+const gesture = Gesture.Tap().onBegin(() => {
+  console.log(_WORKLET);
+});
+```
+
+And here are some examples that won't:
+
+```jsx
+const gesture = Gesture.Tap();
+gesture.onBegin(() => {
+  console.log(_WORKLET);
+});
+```
+
+```jsx
+const callback = () => {
+  console.log(_WORKLET);
+};
+const gesture = Gesture.Tap().onBegin(callback);
+```
+
+```jsx
+const callback = () => {
+  console.log(_WORKLET);
+};
+const gesture = Gesture.Tap();
+gesture.onBegin(callback);
+```
```

Comment:
you should also write here that adding `"worklet";` directive inside the function makes it work

Reply:
done in [aa06fa3](https://github.com/software-mansion/react-native-gesture-handler/pull/3274/commits/aa06fa33641796612b0785b908b13dc6a11e12a1)